operations.py

```diff
@@ -2,7 +2,22 @@
 
 from automaton import Automaton, State
 
-def union(automaton1: Automaton, automaton2: Automaton) -> Automaton:
+import queue
+
+EPSILON_SYMBOL = "$"
```

Comment:
There is no need to re-declare this constant, this can be imported from automaton.py:

```python
from automaton import EPSILON_SYMBOL
```